packages/sdk/src/marketplaces/LooksRare.ts

```diff
@@ -151,6 +154,24 @@ export interface ContractReceipt {
   chainId: number;
 }
 
+interface CollectionResult {
```

Comment:
Not in this PR, but in the future it would be great to create separate folders for marketplaces and put certain stuff into files next to `/marketplaces/index.ts`, because looksRare file is getting very chonky, with all these types and functions.